taui/src/components/route-card.js

```diff
@@ -102,7 +102,7 @@ class RouteCard extends React.PureComponent<Props> {
                   &nbsp;
                   {t('NeighborhoodDetails.FromOrigin')}
                 </span>
-                &nbsp;
+                {' '}
```

Comment:
Neat. I wasn't sure how to expect this to be turned into HTML but it just ends up as a "whitespace" node in the inspector.